snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: charmed-openstack-exporter
-base: core22
+base: core24
```

Comment:
question: do the tests depend on using the new core? Could we reserve this change for a separate PR instead?

Reply:
No, you're right, this should be a separate PR along with the platforms stuff